### PR TITLE
feat lyric song lovely added

### PR DESCRIPTION
**Pull request by abrilmunozc**

### CODE REVIEW

File: 03/index.html [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    <footer>
+        <h6>
+            <p>Fuente: LyricFind</p>
+            Compositores: Billie Eilish O'Connell / Finneas Baird O'Connell / Khalid Robinson
+            Letra de lovely  
+            <p>© Kobalt Music Publishing Ltd., Sony/ATV Music Publishing LLC, Universal Music Publishing Group</p> 
+        </h6>
+    </footer>
+    <div></div>
+    </main>
```
Comment (eamzea) [inline 03/index.html:77]: This doesn't have a proper indentation

File: 03/index.html [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+    </article>
+    <footer>
+        <h6>
+            <p>Fuente: LyricFind</p>
+            Compositores: Billie Eilish O'Connell / Finneas Baird O'Connell / Khalid Robinson
+            Letra de lovely  
+            <p>© Kobalt Music Publishing Ltd., Sony/ATV Music Publishing LLC, Universal Music Publishing Group</p> 
+        </h6>
+    </footer>
+    <div></div>
```
Comment (eamzea) [inline 03/index.html:76]: Why this div is empty?

File: 03/index.html [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+        Whoa, whoa
+        Hello, welcome home
+    </article>
+    <footer>
+        <h6>
+            <p>Fuente: LyricFind</p>
+            Compositores: Billie Eilish O'Connell / Finneas Baird O'Connell / Khalid Robinson
+            Letra de lovely  
+            <p>© Kobalt Music Publishing Ltd., Sony/ATV Music Publishing LLC, Universal Music Publishing Group</p> 
+        </h6>
```
Comment (eamzea) [inline 03/index.html:74]: Paragraphs inside a heading tag?

File: 03/index.html [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+        Wanna feel alive, outside I can't fight my fear
+        Isn't it lovely, all alone?
+        Heart made of glass, my mind of stone
+        Tear me to pieces, skin to bone
+        Hello, welcome home
+        Whoa, yeah
+        Yeah, ah
+        Whoa, whoa
+        Hello, welcome home
+    </article>
```
Comment (eamzea) [inline 03/index.html:67]: This article just have one text tag inside and there're a lot of text

File: 03/index.html [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+        <p>
+    Thought I found a way
+    Thought I found a way out (found)
+    But you never go away (never go away)
+    So I guess I gotta stay now
+    Oh, I hope some day I'll make it out of here
+    Even if it takes all night or a hundred years
+    Need a place to hide, but I can't find one near
+    Wanna feel alive, outside I can't fight my fear
+        </p>
```
Comment (eamzea) [inline 03/index.html:35]: Bad indentation

File: 03/index.html [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="Song lovely by Billie Eilish">
+    <meta name="keywords" content="billi, eilish, lovely, song">
+    <title>Lovely by Billie Eilish (lyric)</title>
+</head>
+    <body>
+    <main>
+    <header>
+        <h1>
+        lovely
```
Comment (eamzea) [inline 03/index.html:14]: Bad indentation

**Review by soyedgarzea — CHANGES_REQUESTED** [review]
Review the indentation

**Review by soyedgarzea — APPROVED** [review]

**Pull request merged**